lib/ws.c

```diff
@@ -296,7 +296,7 @@ static CURLcode ws_decode(struct Curl_easy *data,
     *oleft = 0;             /* bytes yet to come (for this frame) */
   }
 
-  infof(data, "WS: received %zu bytes payload (%zu left, buflen was %zu)",
+  infof(data, "WS: received %Ou bytes payload (%Ou left, buflen was %zu)",
```

Comment:
there's CURL_FORMAT_CURL_OFF_TU. Ou looks cleaner though.
```suggestion
  infof(data, "WS: received %" CURL_FORMAT_CURL_OFF_TU " bytes payload (%"
        CURL_FORMAT_CURL_OFF_TU " left, buflen was %zu)",
```

Reply:
Yeah, when we know it is our own printf() I think we can use Ou, while as if we use an "external" printf we must use `CURL_FORMAT_CURL_OFF_TU`